src/extensions/nuget/Microsoft.DotNet.UpgradeAssistant.Extensions.NuGet/ProjectNuGetReferences.cs

```diff
@@ -54,7 +54,7 @@ public NugetPackageFormat PackageReferenceFormat
             }
         }
 
-        private string? GetPackagesConfigPath() => _project.FindFiles("packages.config", ProjectItemType.Content).FirstOrDefault();
+        private string? GetPackagesConfigPath() => _project.FindFiles("packages.config", ProjectItemType.None).FirstOrDefault();
```

Comment:
Should we just find the file by name without any other constraints?

Reply:
Yea, I went back and forth on that. I went this route because in all the examples I looked at, packages.config was 'None' but it probably doesn't hurt to find it regardless of item type. I'll do that instead.